src/config/configTypes.ts

```diff
@@ -13,6 +13,9 @@ export type RedisConfig = {
   password?: string
   commandTimeout?: number
   connectTimeout?: number
+  maxRetriesPerRequest?: number | null
+  enableReadyCheck?: boolean
+  lazyConnect?: boolean
```

Comment:
@kibertoad I added 2 more parameters that we are using in Flow, so that we are not blocked from using `RedisConfig` in cases when those are needed